cql2pgjson/src/test/java/org/z3950/zing/cql/cql2pgjson/Cql2SqlUtilTest.java

```diff
@@ -76,10 +76,29 @@ public Object cql2likeParams() {
 
   @Test
   @Parameters(method = "cql2likeParams")
-  public void cql2like(String cql, String sql) {
+  public void quotedString(String cql, String sql) {
     assertThat(Cql2SqlUtil.cql2like(cql), is(sql));
   }
+  public Object quotedStringParams() {
+    return params(
+        "           ",
+        "'     ''   ",
+        "a     a    ",
+        "*     %    ",
+        "?     _    ",
```

Comment:
Both * and ? test cases are wrong because such an unmasked CQL wildcard cannot be used in an SQL string constant. Should throw an exception.